pkg/logservice/service.go

```diff
@@ -126,7 +126,8 @@ func NewService(
 	// TODO: check and fix all these magic numbers
 	codec := morpc.NewMessageCodec(mf,
 		morpc.WithCodecPayloadCopyBufferSize(16*1024),
-		morpc.WithCodecEnableChecksum())
+		morpc.WithCodecEnableChecksum(),
+		morpc.WithCodecMaxBodySize(int(cfg.MaxMessageSize)))
```

Comment:
use cfg.RPC to config max message, and cn/dn/log has the same config field

Reply:
have removed it.

Reply:
have changed it.